test/bin/scenario.sh

```diff
@@ -137,19 +143,21 @@ sos_report_for_vm() {
     # version.
     cat - >/tmp/sos-wrapper.sh <<EOF
 #!/usr/bin/env bash
-if [ -f /usr/bin/microshift-sos-report ]; then
-    sudo /usr/bin/microshift-sos-report
+if ! hash sos ; then
+    touch /tmp/sosreport-command-does-not-exist
+elif [ -f /usr/bin/microshift-sos-report ]; then
+    /usr/bin/microshift-sos-report || touch /tmp/sosreport-command-failed
 else
-    sudo chmod +x /tmp/microshift-sos-report.sh
-    sudo PROFILES=network,security /tmp/microshift-sos-report.sh
+    chmod +x /tmp/microshift-sos-report.sh
+    PROFILES=network,security /tmp/microshift-sos-report.sh || touch /tmp/sosreport-command-failed
```

Comment:
Is `touch /tmp/sosreport-command-failed` checked in another script?

Reply:
It's copied to the CI logs using the same `/tmp/sosreport-*` pattern.
So, if sos collection fails, it does not fail the CI job, but we'll see a special file created in the logs

Reply:
The ideas is - never fails CI job because of sos collection errors.

Reply:
We're collecting the logs from this script already. Maybe instead of creating separate files we could just log clear messages?